### DISCUSSION TITLE
How to ignore !**/generated/*.ts

### DISCUSSION THREAD

**Discussion by boringplay** [biomejs/biome#1491] [Q&A] [answered]
When I use ignore config like this `!**/generated/*.ts`, it will cause error: `Couldn't parse the !**/generated/*.ts, reason: recursive wildcards must form a single path component`.

How to ignore those files that are not matching `**/generated/*.ts`?

**Comment by Conaclos** [comment]
We currently have a regression with ignore pattern in the last version (v1.5). We are working on a fix.

**Comment by ematipico** [comment, marked as answer]
Our globs don't support git globs. Even though we have some regressions we're trying to fix, that pattern probably won't work.
